uniflow/op/extract/split/recursive_character_splitter.py

```diff
@@ -11,7 +11,7 @@
 class RecursiveCharacterSplitter(Op):
     """Recursive character splitter class."""
 
-    default_separators = ["\n\n", "\n", " ", ""]
+    default_separators = ["\n\n", "\n", ".", " ", ""]
```

Comment:
shall we use ". " rather than "."? since the later may separate decimals like "0.9".

Reply:
also do we need to add the "." back after the split?

Reply:
+1

Reply:
Make sense! `". "` is better. I think there is no need to add `"."` again.